### ISSUE TITLE
Build error in Xcode 11

### ISSUE THREAD

**Issue by ataravati** [yaslab/CSV.swift#100]
I get the following build error in Xcode 11.

>sed: ./Pods/CSV.swift: in-place editing only works for regular files
Command PhaseScriptExecution failed with a nonzero exit code

**Comment by yaslab** [comment]
Are you editing the source code (.swift file) using the `sed` command in the build phase script? If so, you might be able to build by excluding the `Pods` directory from source code search.

By the way, the reason why the `sed` command fails is because `./Pods/CSV.swift` is a directory, not a file.

**Comment by ataravati** [comment]
> Are you editing the source code (.swift file) using the `sed` command in the build phase script? If so, you might be able to build by excluding the `Pods` directory from source code search.
> 
> By the way, the reason why the `sed` command fails is because `./Pods/CSV.swift` is a directory, not a file.

No, I'm not editing the source code. As soon as I add CSV.swift to my Podfile, I get that error when trying to build.

**Comment by yaslab** [comment]
Hmm, when I created a new Xcode project, there was no `PhaseScriptExecution` command. . .

When `pod install` is executed, two `PhaseScriptExecution` commands, `[CP] Check Pods Manifest.lock` and `[CP] Embed Pods Frameworks` were added by CocoaPods. These are working properly.

----

The environment I tried is as follows.

- Xcode 11.2
- CocoaPods 1.8.4
- macOS 10.15.1

**Comment by ataravati** [comment]
Here's my environment:

- Xcode 11.1
- CocoaPods 1.8.4
- macOS 10.15.1

**Comment by ataravati** [comment]
I upgraded Xcode to 11.2 and I still have the same issue.
